charts/icm-as/values.yaml

```diff
@@ -249,6 +250,7 @@ mssql:
           # - mfsymlinks
           # - cache=strict
           # - actimeo=30
+          reclaimPolicy: Delete
```

Comment:
why should the default reclaimPolicy be Delete? (for the sites folder)

Reply:
Because that's the default value which is already set. With "Retain" you maybe would get a problem with hanging volumes. Someone should explicitly decide himself for having them and then take care about them.
Also for local development it's easier to not take care about hanging volumes. Relevant files are still there after an automatic volume deletion. This lasts for sites and the db.
I talked to OPS and they (for now) also prefer the option just to be able to change this value via `values.yaml`. Maybe after some time we could ask them for whether they like to have "Retain" or "Delete" defaultwise .

Reply:
please use "Delete" on tests and "Retain" for production (default)

Reply:
did it